src/components/Checkbox/index.js

```diff
@@ -116,4 +54,35 @@ class Checkbox extends Component {
   }
 }
 
-export default Checkbox
+export default styled(Checkbox)`
+  input[type='checkbox'] {}
+  input[type='checkbox']:checked {}
+  input[type='checkbox']:checked + .eh-checkbox-icon {
+    color: #1b1f24;
+    background-color: #add960;
+    border-color: #add960;
+  }
+  &.disabled {
+    color: #646e7b;
+    cursor: not-allowed;
+  }
+  &.disabled .eh-checkbox-icon {
+    border-color: #434b56;
+  }
+  &.disabled input[type='checkbox'] {}
+  &.disabled input[type='checkbox']:checked {}
+  &.disabled input[type='checkbox']:checked + .eh-checkbox-icon {
+    background-color: #434b56;
+    border-color: #434b56;
+  }
+  .eh-checkbox-icon {
+    width: 21px;
+    height: 21px;
```

Comment:
@DrShpongle would be good to use tachyons wherever possible instead of hard-coded numbers so we stick to the tachyons relative 1-6  number system